pkg/build/api/v1beta1/conversion_test.go

```diff
@@ -24,6 +24,12 @@ func TestSTIBuildStrategyConversion(t *testing.T) {
 	if actual.Image != oldVersion.BuilderImage {
 		t.Errorf("expected %v, actual %v", oldVersion.BuilderImage, actual.Image)
 	}
+	if actual.From.Name != oldVersion.From.Name {
```

Comment:
don't you need to add a From to one of these?  otherwise you're just comparing nil to nil.